utils/utils_epd.py

```diff
@@ -0,0 +1,146 @@
+# This file is released under MIT licence, see file LICENSE.
```

Comment:
(name of the file) does "epd" stand for expected persistence diagram?

Reply:
yes